README.md

```diff
@@ -25,15 +76,8 @@ flow kind of approach. Development version should be done against the
 `main` branch and then merged to `release` for release.
 (https://guides.github.com/introduction/flow/)
 
-**Kindly cite this work**. 
-
-```
-  list(title = "miaTime R package", 
-       author = list(
-       list(given = "Yagmur", family = "Simsek", role = NULL, email = "yagmur.simsek.98@gmail.com", comment = NULL), 
-       list(given = "Leo", family = "Lahti", role = NULL, email = NULL, comment = NULL)), url = "microbiome.github.io/miaTime", 
-       note = "Version 0.1.2")
-```
+**Kindly cite this work**. For citation details, see R command
+  list(title = "miaTime R package", author = list(list(given = "Yagmur", family = "Simsek", role = NULL, email = "yagmur.simsek.98@gmail.com", comment = NULL), list(given = "Leo", family = "Lahti", role = NULL, email = NULL, comment = NULL)), url = "microbiome.github.io/miaTime", note = "Version 0.1.3").
```

Comment:
Does this render correctly?

Reply:
I think it doesn't look as expected. I will replace this one with `r citation("miaTime")`